next_generation/pint_server/models.py

```diff
@@ -108,11 +108,5 @@ class MicrosoftRegionMapModel(Base, PintBase):
 class VersionsModel(Base, PintBase):
     __tablename__ = 'versions'
 
-    amazonservers = Column(Numeric, nullable=False, primary_key=True)
-    amazonimages = Column(Numeric, nullable=False)
-    googleservers = Column(Numeric, nullable=False)
-    googleimages = Column(Numeric, nullable=False)
-    oracleimages = Column(Numeric, nullable=False)
-    microsoftservers = Column(Numeric, nullable=False)
-    microsoftimages = Column(Numeric, nullable=False)
-    alibabaimages = Column(Numeric, nullable=False)
+    tablename = Column(String(50), primary_key=True)
```

Comment:
Here you have the length as 50 but in https://gitlab.suse.de/public-cloud-team/pint-data/-/merge_requests/68 you have it as 100...

```suggestion
    tablename = Column(String(100), primary_key=True)
```

Reply:
good catch!